lwallet/Main.hs

```diff
@@ -128,6 +131,13 @@ Avaliable commands:
                                      e is current epoch.
    add-key-pool <N>               -- add key from intial pool
    add-key <file>                 -- add key from file
+
+   addr-distr <N> boot
+   addr-distr <N> [<M>:<coinPortion>]+
+                                  -- address of pk <N> (encoded in base58) with provided distribution, where <M> is
+                                  -- stakeholder id (pk hash), and the coin portion can be a coefficient in [0..1]
+                                  -- or a percentage (ex. 42%)
```

Comment:
It's not clear what this command does. Neither from name nor from description. I had no idea until I read the code with its behaviour. Now I see that it **prints** the address and that's it.
Other commands' names are verbs, so it's clearer what they do.